go/libraries/doltcore/sqle/mergeable_indexes_setup_test.go

```diff
@@ -134,24 +134,30 @@ func customRange(tpl1, tpl2 types.Tuple, bt1, bt2 sql.RangeBoundType) *noms.Read
 		if tupleIndex%2 == 0 {
 			return false, nil
 		}
-		nrc = append(nrc, columnBounds{
-			lowerbound: bound{
-				equals:   bt1 == sql.Closed,
-				infinity: false,
-				val:      tupleVal,
-			},
-		})
+		if bt1 == sql.Closed {
+			nrc = append(nrc, columnBounds{
+				boundsCase: boundsCase_greaterEquals_infinity,
+				lowerbound: tupleVal,
+			})
+		} else {
+			nrc = append(nrc, columnBounds{
+				boundsCase: boundsCase_greater_infinity,
+				lowerbound: tupleVal,
+			})
+		}
 		return false, nil
 	})
 	_ = tpl2.IterFields(func(tupleIndex uint64, tupleVal types.Value) (stop bool, err error) {
 		if tupleIndex%2 == 0 {
 			return false, nil
 		}
 		idx := (tupleIndex - 1) / 2
-		nrc[idx].upperbound = bound{
-			equals:   bt2 == sql.Closed,
-			infinity: false,
-			val:      tupleVal,
+		if bt2 == sql.Closed {
+			nrc[idx].boundsCase += 1
```

Comment:
Might add a note to see comment where this happens in production code, it's pretty obtuse